.prettierrc

```diff
@@ -1,7 +1,6 @@
+
```

Comment:
I guess Prettier doesn't work properly in this project. Please configure it properly. As an example you can take [this commit](https://github.com/soft-bee/dental-chat-app/commit/4b1560a9154d093baf5b4fe021a9c18b4f05a652) from `dental-chat-app` repository.

In a6be5725ee58e546484a665700eb06dd81f474a1 commit I pushed additional settings to VS Code that will help both of you in using VS Code, ESLint and Prettier together.